code/lib/client-api/src/ClientApi.ts

```diff
@@ -50,93 +49,61 @@ const warningAlternatives = {
   addDecorator: `Instead, use \`export const decorators = [];\` in your \`preview.js\`.`,
   addParameters: `Instead, use \`export const parameters = {};\` in your \`preview.js\`.`,
   addLoaders: `Instead, use \`export const loaders = [];\` in your \`preview.js\`.`,
+  addArgs: '',
+  addArgTypes: '',
+  addArgsEnhancer: '',
+  addArgTypesEnhancer: '',
+  addStepRunner: '',
+  getGlobalRender: '',
+  setGlobalRender: '',
 };
 
-const warningMessage = (method: keyof typeof warningAlternatives) =>
-  deprecate(
-    () => {},
-    dedent`
-  \`${method}\` is deprecated, and will be removed in Storybook 7.0.
-
-  ${warningAlternatives[method]}
-
-  Read more at https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addparameters-and-adddecorator).`
-  );
-
-const warnings = {
-  addDecorator: warningMessage('addDecorator'),
-  addParameters: warningMessage('addParameters'),
-  addLoaders: warningMessage('addLoaders'),
-};
-
-const checkMethod = (method: string, deprecationWarning: boolean) => {
+const checkMethod = (method: keyof typeof warningAlternatives) => {
   if (global.FEATURES?.storyStoreV7) {
     throw new Error(
       dedent`You cannot use \`${method}\` with the new Story Store.
 
-      ${warningAlternatives[method as keyof typeof warningAlternatives]}`
+      ${warningAlternatives[method]}`
     );
   }
 
   if (!singleton) {
     throw new Error(`Singleton client API not yet initialized, cannot call \`${method}\`.`);
   }
-
-  if (deprecationWarning) {
-    warnings[method as keyof typeof warningAlternatives]();
-  }
-};
-
-export const addDecorator = (
-  decorator: DecoratorFunction<AnyFramework>,
-  deprecationWarning = true
-) => {
-  checkMethod('addDecorator', deprecationWarning);
-  singleton.addDecorator(decorator);
-};
```

Comment:
Did you mean to remove this in this PR?

Reply:
This looks like a mistake to me too

Reply:
We should figure out how to test this in CI, since this kind of code is going to be around until 8.0 and we're currently unprotected. Added to linear.

Reply:
I'm going to revert this, because the storyStore V6 (that we are NOT deprecating yet) still uses these.

But i will remove the public-api in the renderers for these methods, and also remove the other deprecated apis there.

Reply:
We need an automigration that at the very least checks for the existence of `addDecorator` etc. and either upgrades it for them, or at least tells them they need to upgrade by hand. I've also added a Linear issue for this.

Reply:
We added a automigration